script.module.enum34/lib/enum/PKG-INFO

```diff
@@ -0,0 +1,62 @@
+Metadata-Version: 1.1
```

Comment:
remove

Reply:
This file was provided by the original author Ethan Furman
Will removing it break the enum package?

Reply:
no this is only needed to packages for pypi

Reply:
OK